### PR TITLE
New Keyboard: korno65

### PR DESCRIPTION
**Pull request by Jordan-Ramos**
## Description

New keyboard: korno65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/korno65/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# korno65
+
+Korno65
+
+* Keyboard Maintainer: [Jordan-Ramos](https://github.com/Jordan_Ramos)
+* Hardware Supported: PCB v3
+* Hardware Availability: https://eschit.com/products/gb-korno65s-r2-v3
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/korno65/readme.md:9]: Could you add instructions on how to enter the bootloader?  Eg, a hardware reset button, etc.

Comment (Jordan-Ramos) [reply keyboards/korno65/readme.md:9]: added instructions here: https://github.com/qmk/qmk_firmware/pull/13123/commits/f972304a31ecc20cf739c5066b3bcc2f5a7a2a8e

**Review by drashna — APPROVED** [review]

File: keyboards/korno65/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/korno65/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/korno65/config.h [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x1688
+#define PRODUCT_ID   0x1688
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Jordan-Ramos
+#define PRODUCT      korno65
```
Comment (drashna) [inline keyboards/korno65/config.h:27]: This vender ID is in use, and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x1688&p=&n=

```suggestion
#define VENDOR_ID    0xB6C7
#define PRODUCT_ID   0x7527
#define DEVICE_VER   0x0001
#define MANUFACTURER Jordan-Ramos
#define PRODUCT      korno65
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**